### ISSUE TITLE
NTR: Tropheryma whippelii PCR positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8775]
New term request Tropheryma whippelii PCR positive (segal_190605215407): Polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000802
- label: Positive bloodstream Tropheryma whippelii nucleic acid test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of Tropheryma whippelii in the blood circulation by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8775
- synonyms:
